lib/srv/desktop/rdp/rdpclient/src/vchan.rs

```diff
@@ -27,24 +27,30 @@ use std::io::{Cursor, Read};
 /// See https://docs.microsoft.com/en-us/openspecs/windows_protocols/ms-rdpbcgr/343e4888-4c48-4054-b0e3-4e0762d1993c
 /// for more information about chunks.
 pub struct Client {
+    // size_limit is the maximum size of an RDP
```

Comment:
nitpick: can we clarify that's the size limit is in the bytes?

Reply:
77cc8fc8dde8178d9706aaa1e23458a89f982b1d